### DISCUSSION TITLE
Video Buffering - Delay Play Start

### DISCUSSION THREAD

**Discussion by ThePostman111** [immich-app/immich#1891] [General]
Hi,

I have Immich installed on my Synology 920+ and am accessing through DDNS, Synology appears to be having issues with this where speeds are quite slow (not an Immich issue this affects other software and a number of other users as well).

As a result, videos almost immediately buffer when accessed this way, is there a way to delay them from playing until a certain % is buffered (or even the whole thing, I see this was changed in issue 208 to allow for playing whilst buffering)?

Cheers

**Comment by alextran1502** [comment]
Is this the behavior you want in the detail view? i.e clicking on a video?

**Reply by ThePostman111** [reply]
Hi,

I do apologise for the delayed reply, I completely missed the notification for your response.

Yes, the delayed play option would ideally be on the detailed view if at all possible.

I relise that this is a bit of a niche case so it's totally understandable if this is not possible.

Thanks

**Reply by poblabs** [reply]
I don't think this is a niche case. I have the same issue. I've tried QoS, traffic shaping, etc, and videos keep buffering. Would be great to have an option to disable auto-play or to have a delayed auto-start once a % is pre-loaded

**Reply by Thue77** [reply]
I would find this very useful as well

**Reply by eerojaaskelainen** [reply]
A useful feature indeed. Currently the images are working excellent, but videos are quite pain to look over Internet.

**Comment by TheD3vilsAdvocate** [comment]
Kinda strange this hasn't been implemented yet. A mere 5 second delay would probably improve the experience quite a bit.

**Reply by kao-byte** [reply]
Yes, I totally agree
